src/screens/SessionsScreen.tsx

```diff
@@ -1,15 +1,1470 @@
+  navigation?: NativeStackScreenProps<
+    ParamListBase,
+    screen_names.SESSIONS,
+    undefined
+  >["navigation"];
+  dates: DateToggleListProps;
+  sessions: SessionsVerticalListProps;
 };
 
-export default SessionsScreen;
\ No newline at end of file
+const SessionsScreen = (props: SessionsScreenProps) => {
+  const { schedule } = useAppSelector((state) => state.schedule);
+  const [dates, setDates] =
+    useState<Pick<DateToggleListProps, "items"> | undefined>();
+  const [mySessions, setMySessions] = useState<boolean>(false);
+  const [sessions, setSessions] = useState<{ items: Session[] } | undefined>();
+  const [selectedDate, setSelectedDate] = useState<string | undefined>();
+  // const { selectedDay } = useAppSelector((state) => state.schedule.selectedDay);
+  const dispatch = useAppDispatch();
+
+  useEffect(() => {
+    dispatch(setSchedule(sampleScheduleData));
+  }, []);
+
+  useEffect(() => {
+    extractDatesFromSchedule();
+  }, [schedule, selectedDate]);
+
+  useEffect(() => {
+    if (selectedDate && schedule) {
+      setSessions({ items: schedule.data[selectedDate] });
+    }
+  }, [selectedDate]);
+
+  function extractDatesFromSchedule() {
+    if (schedule) {
+      const keys = Object.keys(schedule.data);
+      const dates = keys.map((key, index) => {
+        return {
+          date: `${new Date(key).getDate()}th`,
+          day: `Day ${index + 1}`,
+          fullDate: key,
+          selected: key === selectedDate,
+        };
+      });
+      setDates({ items: dates });
+      if (!selectedDate) {
+        setSelectedDate(keys[0]);
+      }
+    }
+  }
+
+  function onValueChange(value: boolean) {
+    setMySessions(value);
+  }
+
+  function onDayChange(day: string) {
+    setSelectedDate(day);
+  }
+
+  return (
+    <View>
+      <SafeAreaView style={styles.paddingVertical}>
+        <MainHeader />
+        <ScrollView>
+          <View style={{ paddingBottom: 20 }}>
+            <View style={styles.belowHeader}>
+              {dates && <DateToggleList {...dates} onChange={onDayChange} />}
+              <View style={styles.switchHolder}>
+                <SwitchWithIcons
+                  value={mySessions}
+                  onValueChange={onValueChange}
+                  trackColor={{ true: "#FF6E4D", false: "#20201E" }}
+                  thumbColor={{ true: "#FFFFFF", false: "#FFFFFF" }}
+                  iconColors={{ true: "#20201E", false: "#FF6E4D" }}
+                />
+                <Text style={styles.belowSwitch}>My Sessions</Text>
+              </View>
+            </View>
+            <Text style={styles.title}>All Sessions</Text>
+            {sessions && <SessionsVerticalList {...sessions} />}
+          </View>
+        </ScrollView>
+      </SafeAreaView>
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
+  paddingVertical: {
+    paddingVertical: 20,
+  },
+  title: {
+    fontFamily: fonts.MONTSERRAT_BOLD,
+    fontSize: 18,
+    color: "rgba(0, 12, 235, 1)",
```

Comment:
Uhm could you define the `rgba(0, 12, 235, 1)` on `constants/colors.ts` file instead?